docs/index.md

```diff
@@ -351,17 +338,34 @@ The following limitations apply when presenting Health Card as QR codes, rather
 
 
-### Numerical Encoding
+## Creating a QR code (or a set of QR codes) from a Health Card JWS
 
-When printing or displaying a Health Card as a QR code, the the JWS string value SHALL be represented as two segments:
+### Chunking
 
-1. A segment encoded with `bytes` mode consisting of the fixed string `shc:/`
+Commonly, Health Cards will fit in a single V22 QR code.  Any JWS longer than 1195 characters is split into "chunks" of length 1191 or smaller; each chunk is then encoded as a separate QR code to ensure ease of scanning.  Each chunk is numerically encoded and prefixed with an ordinal as well as the total number of chunks required to re-assemble the JWS, as described below.
 
+Consumers of the QR Code SHOULD allow for scanning the multiple QR Codes in any order. Once the full set is scanned, the JWS is assembled and validated. 
+
+### Encoding Chunks as QR Codes
+
+When printing or displaying a Health Card using QR codes, let "N" be the total number of chunks required, and let "C" be a variable indicating the index of the current chunk. Each chunk of the JWS string value is represented as a QR with two data segments:
+
+1. A segment encoded with `bytes` mode consisting of 
+    * the fixed string `shc:/` 
+    * plus (only if more than one chunk is required)
+        *  decimal representation of "C" (e.g., `1` for the first chunk, `2` for the second chunk, and so on)
+        *  plus the fixed string `/`
+        *  plus decimal representation of "N" (e.g., `2` if there are two chunks in total, `3` if there three chunks in total, and so on)
+        *  plus the fixed string `/`
 2. A segment encoded with `numeric` mode consisting of the characters `0`-`9`. Each character "c" of the JWS is converted into a sequence of two digits as by taking `Ord(c)-45` and treating the result as a two-digit base ten number. For example, `'X'` is encoded as `43`, since `Ord('X')` is `88`, and `88-45` is `43`. (The constant "45" appears here because it is the ordinal value of `-`, the lowest-valued character that can appear in a compact JWS. Subtracting 45 from the ordinal values of valid JWS characters produces a range between 00 and 99, ensuring that each character of the JWS can be represented in exactly two base-10 numeric digits.)
 
 (The reason for representing Health Cards using Numeric Mode QRs instead of Binary Mode (Latin-1) QRs is information density: with Numeric Mode, 20% more data can fit in a given QR, vs Binary Mode. This is because the JWS character set conveys only log_2(65) bits per character (~6 bits); binary encoding requires log_2(256) bits per character (8 bits), which means ~2 wasted bits per character.)
 
-When reading a QR code, scanning software can recognize a SMART Health Card from the `shc:/` prefix. Stripping this prefix and decoding the remaining pairs of numerals yields a JWS.
+For example:
+* a single chunk might produce a QR code like `shc:/56762909524320603460292437404460<snipped for brevity>`
```

Comment:
Thoughts on always including `C` and `N` indicators (chunk metadata), even when the set only contains a single QR code? So

```suggestion
* a single chunk might produce a QR code like `shc:/1/1/56762909524320603460292437404460<snipped for brevity>`
```

While Health Card verifiers could assume the lack of chunk metadata to indicate a set of 1, I think the 4 characters is worth it for the explicitness it provides.

Reply:
I think the one-card case is still designed to be the common case. I don't feel super strongly here but this comes at a cost of 4 characters from the JWS, which can mean be the difference between fitting and not.

Reply:
Definitely, the one-QR code case should the the common case! The UX will be so much better with a single code. My $0.02 -- we should optimize for the single QR code scenario, b/c it'll be the most common. This means, leaving out N, C, and the delimiters when there's only a single code.

Reply:
Sounds good to me – just wanted to surface. sounds like @isaacvetter and team have gotten deeper into the implementation side of splitting/reconstruction, so I trust that the complexity isn't too much and agreed that we should optimize for the single QR code scenario.